src/index.html

```diff
@@ -10,7 +10,52 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form
+      action="big"
+      method="get"
+      class="form search__big"
+      data-qa="big"
+    >
+
+      <img
+        src="./images/Search.svg"
+        alt="magnifying glass"
+        class="icon icon_big"
+      >
+
+      <input
+        type="text"
+        class="input input__big"
```

Comment:
```suggestion
        class="input"
```
Please follow the approach from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) p2. It would be better to create one common class, and just override properties for small input. Same for the other elements